apis/nucleus/src/components/listbox/components/ListBoxSearch.jsx

```diff
@@ -6,9 +6,12 @@ import InstanceContext from '../../../contexts/InstanceContext';
 import useDataStore from '../hooks/useDataStore';
 import { CELL_PADDING_LEFT } from '../constants';
 
+const MAX_SEARCH_LENGTH = 5000; // limitation in Engine
 const TREE_PATH = '/qListObjectDef';
 const WILDCARD = '**';
 
+const sanitizeValue = (val) => val?.substring(0, MAX_SEARCH_LENGTH); // limit length
```

Comment:
Maybe call it `limitValue` or `limitSearchValue` instead to indicate better what it's actually doing, since it's not actually "sanitizing" the string, right?